ncf-api-virtualenv/SOURCES/ncf-api-virtualenv.conf

```diff
@@ -4,7 +4,7 @@
 Alias /ncf-builder /usr/share/ncf/builder
 
 ## Set up a WSGI serving process named ncf_api_flask_app
-WSGIDaemonProcess ncf_api_flask_app threads=5 user=ncf-api-venv
+WSGIDaemonProcess ncf_api_flask_app threads=5 user=ncf-api-venv umask=0007
```

Comment:
The files written by the builder are `ncf-api-venv:rudder`, whereas the content of `ncf/local` is `root:root`.